client/src/views/Simulation/components/ParametersPanel.vue

```diff
@@ -184,7 +174,10 @@
       const yMinMax = [marginY, this.graphHeight() - marginY];
 
       // X & Y Scales
-      const xScale = param => d3.scaleLinear(d3.extent(this.runs, d => d[param]), xMinMax);
+      const xScale = param => {
+        const minMax = d3.extent(runs, d => d[param]) as Iterable<d3.NumberValue>;
+        return d3.scaleLinear(minMax, xMinMax);
```

Comment:
Maybe you want to use `axis` here from `utils/SVGUtil`

Reply:
unfortunately I can't, because `SVGUtils.axis()` uses `d3.nice()` and mess up the whole rounding up to power of 10 I do to display the parameters value in a nice way.